ACTIONS.md

```diff
@@ -0,0 +1,174 @@
+# GitHub Actions setup
+
+Quarkus is built using GitHub actions, with a mix of hosted and self-hosted runners. 
+
+## Setting up a new self-hosted runner on Mac M1 
+
+
+### Non-root user
+
+GitHub actions should not run with administrator privileges. 
+Make an account for the actions runner. *Grant administrator privileges to the account* (we will remove them later).
```

Comment:
Given you have a name for the account later, maybe add it there to be consistent?